packages/block-library/src/search/edit.js

```diff
@@ -436,16 +457,20 @@ export default function SearchEdit( {
 							onUnitChange={ ( newUnit ) => {
 								setAttributes( {
 									width:
-										'%' === newUnit
+										newUnit === '%'
```

Comment:
```suggestion
										'%' === newUnit
```

This change doesn't affect anything, but I don't think it's necessary for this PR, so let's revert the change.